File: src/BukkitPE/event/entity/EntityCombustByEntityEvent.php
<?php
namespace BukkitPE\event\entity;

use BukkitPE\entity\Entity;

class EntityCombustByEntityEvent extends EntityCombustEvent{

	protected $combuster;

	/**
	 * @param Entity $combuster
	 * @param Entity $combustee
	 * @param int    $duration
	 */
	public function __construct(Entity $combuster, Entity $combustee, $duration){
		parent::__construct($combustee, $duration);
		$this->combuster = $combuster;
	}

	/**
	 * @return Entity
	 */
	public function getCombuster(){
		return $this->combuster;
	}

}

File: src/main/java/net/BukkitPE/event/entity/EntityCombustByEntityEvent.java
package net.BukkitPE.event.entity;

import net.BukkitPE.entity.Entity;

/**

 * BukkitPE Project
 */
public class EntityCombustByEntityEvent extends EntityCombustEvent {

    protected Entity combuster;

    public EntityCombustByEntityEvent(Entity combuster, Entity combustee, int duration) {
        super(combustee, duration);
        this.combuster = combuster;
    }

    public Entity getCombuster() {
        return combuster;
    }
}
